incubator/common/pipelines/default/build-task.yaml

```diff
@@ -18,10 +18,10 @@ spec:
       - name: docker-image
         type: image
   steps:
-    - name: assemble-extract-step
+    - name: assemble-extract
       securityContext:
         privileged: true
-      image: appsody/appsody-docker
+      image: appsody/appsody-buildah
       command: ["/bin/bash"]
       args:
         - -c
```

Comment:
@kvijai82 - `appsody-buildah` comes up now with `extract.sh` as opposed to `setupAndRunExtract.sh` in `appsody-docker`